models/hospital.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+const { Model } = require("sequelize");
+module.exports = (sequelize, DataTypes) => {
+  class Hospital extends Model {}
+  Hospital.init(
+    {
+      id: {
+        field: "hospital_uuid",
+        type: DataTypes.UUID,
+        primaryKey: true,
+        autoIncrement: true,
```

Comment:
I took out the ```allowNull: false``` and replaced it with ```autoIncrement: true``` so that we DON'T need to specify an id on creation- instead, Sequelize will let the database create it per the default function configured in the migration.

We'll want to make this change on all the primary key attribute configs.